[2022-12-04T16:14:03.955+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: data_generation.generate_data scheduled__2022-12-04T03:00:00+00:00 [queued]>
[2022-12-04T16:14:03.962+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: data_generation.generate_data scheduled__2022-12-04T03:00:00+00:00 [queued]>
[2022-12-04T16:14:03.963+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2022-12-04T16:14:03.964+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 1
[2022-12-04T16:14:03.966+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2022-12-04T16:14:03.975+0000] {taskinstance.py:1304} INFO - Executing <Task(PythonOperator): generate_data> on 2022-12-04 03:00:00+00:00
[2022-12-04T16:14:03.981+0000] {standard_task_runner.py:55} INFO - Started process 492 to run task
[2022-12-04T16:14:03.985+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'data_generation', 'generate_data', 'scheduled__2022-12-04T03:00:00+00:00', '--job-id', '57', '--raw', '--subdir', 'DAGS_FOLDER/generate_data.py', '--cfg-path', '/tmp/tmp8sqjidxw']
[2022-12-04T16:14:03.988+0000] {standard_task_runner.py:83} INFO - Job 57: Subtask generate_data
[2022-12-04T16:14:04.071+0000] {task_command.py:389} INFO - Running <TaskInstance: data_generation.generate_data scheduled__2022-12-04T03:00:00+00:00 [running]> on host c7dedbf79ea9
[2022-12-04T16:14:04.135+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=data_generation
AIRFLOW_CTX_TASK_ID=generate_data
AIRFLOW_CTX_EXECUTION_DATE=2022-12-04T03:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-12-04T03:00:00+00:00
[2022-12-04T16:14:04.238+0000] {python.py:177} INFO - Done. Returned value was: None
[2022-12-04T16:14:04.253+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=data_generation, task_id=generate_data, execution_date=20221204T030000, start_date=20221204T161403, end_date=20221204T161404
[2022-12-04T16:14:04.277+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2022-12-04T16:14:04.298+0000] {taskinstance.py:2582} INFO - 1 downstream tasks scheduled from follow-on schedule check
